custom_components/yamaha_ynca/button.py

```
@@ -17,7 +17,11 @@
 )
 
 
-async def async_setup_entry(hass: HomeAssistant, config_entry: YamahaYncaConfigEntry, async_add_entities: AddEntitiesCallback):
+async def async_setup_entry(
```

Ruff (0.8.2)

Missing return type annotation for public function `async_setup_entry`

Add return type annotation: `None`

(ANN201)